Dockerfile

```diff
@@ -26,8 +27,8 @@ RUN set -x \
   && true
 
 # install tox
-ENV PYTHON_TOX_VERSION 4.14.0
-ENV PYTHON_TOX_UV_VERSION 1.5.1
+ENV PYTHON_TOX_VERSION=4.14.0
+ENV PYTHON_TOX_UV_VERSION=1.5.1
```

Comment:
Pls, bump tox to 4.14.1 and tox uv to 1.7.0. So it is in sync with tox-requirements.txt

Reply:
updated